### PR TITLE
Make default macro names consistent

### PR DESCRIPTION
**Pull request by adayton1**
Make sure all loop macros start with CARE and end with LOOP so we don't confuse users or cause macro conflicts.

### CODE REVIEW

File: docs/sphinx/tutorial.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Another policy of note is raja_chai_everywhere. This runs the code on both the host and the device and is intended to be used with managed_ptr to keep both the host and device object instance in sync.
+
+Finally, there are fusible policies. Loops can be "fused" together into a single kernel to reduce kernel launch overhead. More details will be forthcoming on this.
+
+These policies can be used with care::forall, which then forwards the arguments to RAJA::forall. CARE also provides convenient macros that can fall back to vanilla for loops if CARE is configured with -DENABLE_LEGACY_CARE=ON. These macros generally start with CARE and end with LOOP. For example, CARE_SEQUENTIAL_LOOP always executes sequentially. CARE_STREAM_LOOP executes on the GPU if GPU_ACTIVE is defined in the compilation unit and nvcc is used to compile the code, otherwise it falls back to executing using OpenMP on the host if OPENMP_ACTIVE is defined in the compilation unit and the correct OpenMP flags are passed to the compiler, otherwise it falls back to running sequentially on the host.
+
```
Comment (dtaller) [inline docs/sphinx/tutorial.rst:24]: I would add a sentence or two like "Below we provide some sample code to demonstrate the CARE concepts described above. Please note that <anything the user should look out for in terms of compiling, running the code, etc. if applicable>."

File: docs/sphinx/tutorial.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+         array[i] = i;
+      } CARE_GPU_LOOP_END
+
+      // Will run in parallel on the host if compiled with the OpenMP flags.
+      // Otherwise will run sequentially on the host.
+      CARE_OPENMP_LOOP(i, 0, length) {
+         array[i] = array[i] * i;
+      } CARE_OPENMP_LOOP_END
+
+      // Will run on the GPU if compiled with nvcc.
```
Comment (robinson96) [inline docs/sphinx/tutorial.rst:48]: probably should comment as compie with nvcc or hip

File: docs/sphinx/tutorial.rst (hunk 1 above)
Comment (adayton1) [reply docs/sphinx/tutorial.rst:24]: Good idea! Done.

File: docs/sphinx/tutorial.rst (hunk 2 above)
Comment (adayton1) [reply docs/sphinx/tutorial.rst]: Good point. Done.

**Comment by adayton1** [comment]
I think all comments have been addressed. Let me know if anything is needed.

File: docs/sphinx/tutorial.rst (hunk 1 above)
Comment (dtaller) [reply docs/sphinx/tutorial.rst:24]: Looks good. Just replace the phrase "nvcc is used" with "nvcc or hipcc is used", similar to Peter's previous comment

File: docs/sphinx/tutorial.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+Another policy of note is raja_chai_everywhere. This runs the code on both the host and the device and is intended to be used with managed_ptr to keep both the host and device object instance in sync.
+
+Finally, there are fusible policies. Loops can be "fused" together into a single kernel to reduce kernel launch overhead. More details will be forthcoming on this.
+
+These policies can be used with care::forall, which then forwards the arguments to RAJA::forall. CARE also provides convenient macros that can fall back to vanilla for loops if CARE is configured with -DENABLE_LEGACY_CARE=ON. These macros generally start with CARE and end with LOOP. For example, CARE_SEQUENTIAL_LOOP always executes sequentially. CARE_STREAM_LOOP executes on the GPU if GPU_ACTIVE is defined in the compilation unit and nvcc is used to compile the code, otherwise it falls back to executing using OpenMP on the host if OPENMP_ACTIVE is defined in the compilation unit and the correct OpenMP flags are passed to the compiler, otherwise it falls back to running sequentially on the host.
```
Comment (dtaller) [inline docs/sphinx/tutorial.rst:23]: Just change "GPU_ACTIVE is defined in the compilation unit and nvcc is used" to "nvcc and hipcc", other than that looks fine.

Comment (adayton1) [reply docs/sphinx/tutorial.rst]: Fixed.

**Pull request merged**